recipes/math-mode

```diff
@@ -0,0 +1,3 @@
+(math-mode :repo "ashok-khanna/math-mode"
+                :fetcher github
+                :files ("math-mode.el"))
```

Comment:
```suggestion
(math-mode :repo "ashok-khanna/math-mode" :fetcher github)
```
This recipe is equivalent and MELPA prefers the default recipe when possible -- it gives you room to grow.